<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Saving and Loading</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="enableDtd">
            <property name="toolTip">
             <string>Not enabled by default since a reference to an external DTD is known to cause problems with some XML parsers.</string>
            </property>
            <property name="text">
             <string>Include &amp;DTD reference in saved maps</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="openLastFiles">
            <property name="text">
             <string>Open last files on startup</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="reloadTilesetImages">
            <property name="text">
             <string>&amp;Reload tileset images when they change</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="safeSaving">
            <property name="toolTip">
             <string>Turn this off if you're having trouble saving your files.</string>
            </property>
            <property name="text">
             <string>Use safe writing of files</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Interface</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Quad color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="Tiled::Internal::ColorButton" name="gridColor"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Grid color:</string>
            </property>
            <property name="buddy">
             <cstring>gridColor</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="wheelZoomsByDefault">
            <property name="text">
             <string>Mouse wheel &amp;zooms by default</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Object line width:</string>
            </property>
            <property name="buddy">
             <cstring>objectLineWidth</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="3">
           <widget class="QDoubleSpinBox" name="objectLineWidth">
            <property name="suffix">
             <string> pixels</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QSpinBox" name="gridFine">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>128</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="languageCombo"/>
          </item>
          <item row="3" column="3">
           <widget class="Tiled::Internal::ColorButton" name="quadColor">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Fine grid divisions:</string>
            </property>
            <property name="buddy">
             <cstring>gridFine</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="4">
           <widget class="QCheckBox" name="openGL">
            <property name="text">
             <string>Hardware &amp;accelerated drawing (OpenGL)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Language:</string>
            </property>
            <property name="buddy">
             <cstring>languageCombo</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Megamix Engine</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="exeLabel">
            <property name="text">
             <string>Executable</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="exePath">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="exeBrowse">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="gmProjectPathLabel">
            <property name="text">
             <string>GMProjectDir</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="gmProjectDir">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="gmProjectBrowse">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="gmProjectPathLabel">
            <property name="text">
             <string>GMProjectFile</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="gmProjectFile">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="gmProjectFileBrowse">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="genTemplatesOutDirLabel">
            <property name="text">
             <string>TemplatesOutDir</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="genTemplatesOutDir">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QToolButton" name="genTemplatesOutDirBrowse">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="defaultCombineTilesCheckBox">
         <property name="text">
          <string>Combine tiles on saved room files by default</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>381</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Theme</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Theme</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>72</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="styleCombo"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="selectionColorLabel">
            <property name="text">
             <string>Selection color:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="Tiled::Internal::ColorButton" name="selectionColor"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Style:</string>
            </property>
            <property name="buddy">
             <cstring>styleCombo</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="Tiled::Internal::ColorButton" name="baseColor"/>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>72</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="baseColorLabel">
            <property name="text">
             <string>Base color:</string>
            </property>
            <property name="buddy">
             <cstring>baseColor</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Updates</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Updates</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="3">
           <widget class="QPushButton" name="checkForUpdate">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Check Now</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QCheckBox" name="autoUpdateCheckBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Automatically check for updates</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lastAutoUpdateCheckLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Enabled Plugins</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListView" name="pluginList"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tiled::Internal::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>enableDtd</tabstop>
  <tabstop>reloadTilesetImages</tabstop>
  <tabstop>openLastFiles</tabstop>
  <tabstop>safeSaving</tabstop>
  <tabstop>languageCombo</tabstop>
  <tabstop>gridColor</tabstop>
  <tabstop>gridFine</tabstop>
  <tabstop>objectLineWidth</tabstop>
  <tabstop>openGL</tabstop>
  <tabstop>styleCombo</tabstop>
  <tabstop>selectionColor</tabstop>
  <tabstop>baseColor</tabstop>
  <tabstop>checkForUpdate</tabstop>
  <tabstop>autoUpdateCheckBox</tabstop>
  <tabstop>pluginList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
